Matrix.org details (#441)
ansible/community: group-community/communication.rst

> What problem are we trying to solving?

Reviewer: `s/solving/solve/`

======================================

> We recognise that although a lot of people know and love IRC it does have some short commings for some people, such as:

Reviewer 2: ```suggestion
We recognise that although a lot of people know and love IRC, it does have some shortcomings for some people, such as:
```

> * Unless you are running a persistent connection (bouncer) you lose history

Reviewer 2: ```suggestion
* Unless you are running a persistent connection (`bouncer <https://en.wikipedia.org/wiki/BNC_(software)>`_) you lose history
```
Not that people think it's another word for doorman :)

* Not everybody likes IRC

> The Ansible Community Working Group is looking at making it easier for people to join the Ansible Discussions

Reviewer 2: ```suggestion
The Ansible Community Working Group is looking at making it easier for people to join the Ansible Discussions.
```

What's the proposed solution?
=============================

We will experiment with using `Matrix.org <https://matrix.org>`_ to allow people to connect to our Freenode channels.

Matrix is an ecosystem for open and interoperable communication.

How can we use Matrix?
======================

> 1. Create a `Matrix.org account <https://matrix.org/beta/#/register>`_

Reviewer 3: swap 
https://matrix.org/beta/#/register
for 
https://riot.im/app/#/register

The best place to register is https://riot.im/app/#/register - we recommend Riot as a client for those unfamiliar with Matrix, but even if the person would like to use another client, many clients do not currently implement the registration flow (and people fall back to Riot).

https://matrix.org/beta/#/register links to a legacy client (now unmaintained)  - it will probably still work, though I can't vouch for the UX.

> If IRC works for your, please continue to use it. You shouldn't notice any difference.

Reviewer 4: If IRC works for `you` (not `your`)

2. Join `ansible on Matrix <https://matrix.to/#/#freenode_#ansible:matrix.org>`_, if you are just starting we suggest riot.im
  

>    a. Sign into `#ansible on riot.im <https://riot.im/app/#/room/#freenode_#ansible:matrix.org>`_ using the credentials you created in step 1

Reviewer 3: If the link above is updated to a Riot link - then you probably won't need to re-sign in

1. Create a `Matrix.org account <https://riot.im/app/#/register>`_
2. Join `ansible on Matrix <https://matrix.to/#/#freenode_#ansible:matrix.org>`_, if you are just starting we suggest riot.im

> 3. Join any of the other channels listed below that you are interested in

Reviewer 4: period instead of comma after 3

Reviewer 4: please ignore this ^ comment, my eyes played tricks on me

> 4 We love feedback, let us know how it's going in `ansible-community <https://matrix.to/#/#freenode_#ansible-community:matrix.org>`_.

Reviewer 4: Need a period after 4

List of all channels
~~~~~~~~~~~~~~~~~~~~

* `#ansible on Matrix <https://matrix.to/#/#freenode_#ansible:matrix.org>`_ - user discussions
* `#ansible-devel on Matrix <https://matrix.to/#/#freenode_#ansible-devel:matrix.org>`_ - Developer discussions
* `#ansible-aws on Matrix <https://matrix.to/#/#freenode_#ansible-aws:matrix.org>`_ - Ansible and Amazon's Cloud (AWS) Discussions
* `#ansible-awx on Matrix <https://matrix.to/#/#freenode_#ansible-awx:matrix.org>`_ - Ansible AWX, the opensource upstream version of Ansible Tower
* `#ansible-azure on Matrix <https://matrix.to/#/#freenode_#ansible-azure:matrix.org>`_ Ansible & Azure
* `#ansible-community on Matrix <https://matrix.to/#/#freenode_#ansible-community:matrix.org>`_ Ansible Community discussions
* FIXME: I'll add the others once people are happy with the structure

What other ways can I join the conversation?
============================================

* The `Ansible Communication page <https://docs.ansible.com/ansible/devel/community/communication.html>`_ lists all the email lists and IRC channels that we have

How do I report feedback?
=========================

> In ``#ansible-community`` `ansible-community on Matrix <https://matrix.to/#/#freenode_#ansible-community:matrix.org>`_.

Reviewer 2: ```suggestion
In ``#ansible-community`` (Freenode), respectively `ansible-community on Matrix <https://matrix.to/#/#freenode_#ansible-community:matrix.org>`_.
```
